sync-server/src/bin/taskchampion-sync-server.rs

```diff
@@ -1,10 +1,21 @@
 #![deny(clippy::all)]
 
-use actix_web::{middleware::Logger, App, HttpServer};
+use actix_web::{middleware, middleware::Logger, App, HttpServer};
 use clap::Arg;
 use taskchampion_sync_server::storage::SqliteStorage;
 use taskchampion_sync_server::Server;
 
+// The `.wrap` method returns an opaque type, meaning that we can't easily return it from
+// functions.  So, we must apply these default headers when the app is created, which occurs both
+// in `main` and in the tests.  To check that those are both doing precisely the same thing, we use
+// a macro.  This is ugly, and will go away when actix-web is no longer the framework in use.
+macro_rules! cache_control_headers {
```

Comment:
Could maybe be named `no_cache_header` or something a bit more explicit? But not important

Reply:
It does add the `Cache-Control` header.  I don't know why I used a plural but it's close enough :)